src/scenes/ProgressReports/EditForm/Steps/SubmitReportStep/TransitionProgressReport.graphql

```diff
@@ -6,5 +6,8 @@ mutation TransitionProgressReport(
     status {
       ...ProgressReportStatus
     }
+    workflowEvents {
```

Comment:
I'd add a comment here saying grab the updated list from the change. Just to be extra clear that this isn't directly needed, but instead to provide and updated cache value for other pages.